src/parser/parser.yy

```diff
@@ -1136,6 +1155,16 @@ type_spec
         $$ = new meta::cpp2::ColumnTypeDef();
         $$->set_type(meta::cpp2::PropertyType::DATETIME);
     }
+    | KW_GEOGRAPHY {
+        $$ = new meta::cpp2::ColumnTypeDef();
+        $$->set_type(meta::cpp2::PropertyType::GEOGRAPHY);
+        $$->set_geo_shape(meta::cpp2::GeoShape::ANY);
+    }
+    | KW_GEOGRAPHY L_PAREN geo_shape_type R_PAREN {
+        $$ = new meta::cpp2::ColumnTypeDef();
+        $$->set_type(meta::cpp2::PropertyType::GEOGRAPHY);
+        $$->set_geo_shape($3);
+    }
```

Comment:
There is no syntax allow user to write GeoGraphy value.

Reply:
There is a factory method named `Geography::fromWKT()` to allow user to construct a Geography value.
In next or, I will add a constructor to construct a Geography value from a Geometry

Reply:
I means writing GeoGraphy in nGQL

Reply:
ST_GeogFromText